provers/blevm/script/src/bin/main.rs

```diff
@@ -106,15 +110,17 @@ async fn main() -> Result<(), Box<dyn Error>> {
     if args.prove {
         println!("Generating proof...");
         let start = Instant::now();
-        let (proof, _) = prover.generate_proof(input).await?;
+        let (proof, vk) = prover.generate_proof(input).await?;
         let duration = start.elapsed();
         println!("Generated proof in {:?}.", duration);
 
-        let proof_bin = bincode::serialize(&proof)?;
-        // Save proof to file
-        println!("Saving proof to proof.bin");
-        fs::write("proof.bin", proof_bin)?;
+        let aggregation_input = AggregationInput { proof, vk };
+        let aggregation_input_bin = bincode::serialize(&aggregation_input)?;
+
+        println!("Saving proof to proof.bin.");
+        fs::write("proof.bin", aggregation_input_bin)?;
         println!("Saved proof.");
+
         return Ok(());
```

Comment:
[question] do we want to save the single block proof in addition to the aggregate proof?

Reply:
The output is a single block proof and verifying key combined, which will act as the input to the aggregation program. We should rename it as proof_with_key.bin or something.

Reply:
I think this can be done later as a separate PR.